docs/docs/guides/sso.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Configuring Single sign-on
+---
+# Configuring Single sign-on
+
+In Infrahub you can configure SSO using either Open ID Connect (OIDC), alternatively you can use OAuth2.
+
+The SSO system on with Infrahub allows for the configuration of one or more identity providers. While most organizations will only use one provider a reason to have two could be that the providers manage different security domains where one of them might be for regular users the other identity provider could be for administrative accounts.
+
+Infrahub supports three different OIDC providers:
+
+* PROVIDER1
+* PROVIDER2
+* GOOGLE
+
+All of them work in the same way the main difference is that the one for Google include some predefined settings that limits the amount of configuration you have to do yourself.
+
+When configuring Infrahub the ways to setup Infrahub using OAuth2 is fairly close to what is required to configure OIDC, though there are some slight differences with regards to the settings you need to have in place, both options are provided below.
```

Comment:
When configuring Infrahub, setting up OAuth2 or OIDC is fairly similar, though there are some slight differences with regards to the settings you need to have in place. Both options are provided below.